packages/contracts/contracts/minter-suite/Minters/MinterSetPriceERC20V5.sol

```diff
@@ -528,6 +524,10 @@ contract MinterSetPriceERC20V5 is ReentrancyGuard, ISharedMinterV0 {
             _isEngineCaches[_coreContract]
         );
         // process payment in either ETH or ERC20
+        ERC20Lib.ProjectCurrencyConfig
```

Comment:
moved for a common diff, originally driven by stack-too-deep error (and this enables block-scoping)